### DISCUSSION TITLE
Unable to pairing CCT5015

### DISCUSSION THREAD

**Discussion by persching1978** [Koenkk/zigbee2mqtt#11717] [Devices]
Hello,
I've bought a Schneider Electric CCT5015-0002 because -0001 was out of stock. Now I've connected it to AC and tried to pair it with zigbee2mqtt, but it's not working. Is it because it's version 0002 and not 0001 like in the devicelist?? 
I've pressed the reset button 3 times short and the LED is blinking, so I think it's in the pairing mode. Then I've started the pairing mode in the zigbee2mqtt UI.

Has anybody connected this device to zigbee2mqtt??

**Comment by persching1978** [comment]
I think I've found the solution: I use a CC2531 Stick and so it's not zigbee 3.0 compatible. I've ordered a 3.0 compatible stick and then I'll try again.

**Comment by bemble** [comment]
Hi @persching1978 , did you get it working? 

I also have a CCT5015-0002, recognized by zigbee2mqtt as CCT5015-0001, but I can't send commands (set lift duration open/close etc).

**Reply by persching1978** [reply]
Yes it's working. I need to use a new coordinator stick and then it works. I used the Sonoff zigbee 3.0 Dongle.

**Reply by bemble** [reply]
I tried with a Sonoff Zigbee 3.0 plus dongle, nothing changed, the module might be faulty. I was just trying this module, I will search for alternatives.

**Comment by dungtri** [comment]
Hi, I have two CCT5015-0002W, recognized by zigbee2mqtt as CCT5015-0001, with a Conbee II stick and it work well now. I got issue to pairing my second module and I solve that with a factory reset :
- Short press (< 0.5 s) the connected push-button switch 3 times and then long press the push-button switch once for >10 s and <15 s to reset the module to factory default mode.
- Then just push-button switch 3 times for pairing again.

**Reply by lippertmarkus** [reply]
@dungtri 
according to the user manual for the -0002W one can configure different lift durations for up/down and the rotation duration for tilting:
<img width="483" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/1463140/b310449b-7809-4ef4-9db4-57b6fc1175d4">

Did you by chance find out which cluster & attributes are used for those options? Asking because the -0001 only hase a single attribute for the lift duration: https://github.com/Koenkk/zigbee-herdsman-converters/blob/f9c6313e517b4d805a75940cb608f98a57338a42/src/devices/schneider_electric.ts#L463

**Reply by lippertmarkus** [reply]
I found the Zigbee device specifications: https://github.com/Koenkk/zigbee2mqtt/discussions/23111
with the attibutes of those new attributes:
<img width="305" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/1463140/0aebc733-4f98-42c5-97c4-866c76906c0a">
